pkg/network/protocols/http/shared_libraries.go

```diff
@@ -230,36 +222,29 @@ func (w *soWatcher) Start() {
 				}
 
 				lib := toLibPath(event.Data)
+				if int(lib.Pid) == thisPID {
```

Comment:
please filter like below with the comment old line 240
`if strings.HasPrefix(libPath, w.procRoot) {`

Reply:
as discuss offline, as the tests are standalone process now, this will not the case anymore